src/views/players/PlayerView/PlayerView.vue

```diff
@@ -224,16 +289,68 @@
 
     isAchieved(a: ProfileAchievement): boolean {
       let b = true;
-      Object.values(this.myAchievements).forEach( (value) => {
-        if (value.title === a.title) {
+      Object.values(this.myAchievements).forEach((value) => {
+        if (!Object.prototype.hasOwnProperty.call(a, 'title')) {
```

Comment:
Why use hasOwnProperty instead of just `if (a.title)`? Same point for later in the file